src/stores/roles/state.ts

```diff
@@ -0,0 +1,8 @@
+import type { Role } from '@/domain/Role'
+import type { QuizAnswers } from '@/domain/QuizAnswer'
+import { type Ref, ref } from 'vue'
+
+const roles: Ref<Role[]> = ref([])
+const quizAnswer: Ref<QuizAnswers> = ref({} as QuizAnswers)
```

Comment:
Maybe this should be in a quiz/state.ts (as the action to initialize the state), no? The Quiz is beyond the role. Do you agree?